mgmtsystem/hawkular.py

```diff
@@ -210,78 +244,131 @@ def wait_vm_stopped(self, vm_name, num_sec):
     def wait_vm_suspended(self, vm_name, num_sec):
         raise NotImplementedError('wait_vm_suspended not implemented.')
 
-    def list_server_deployment(self, **kwargs):
+    def list_server_deployment(self, feed_id=None):
```

Comment:
Only danger here is if a command already passes args to it......it will then break, @mfalesni what do you think about changing the prototype here?